packages/orchestration/README.md

```diff
@@ -346,21 +347,16 @@ const orchestrationClient = new OrchestrationClient({
     ],
     defaults: {}
   },
-  grounding: {
-    type: 'document_grounding_service',
-    config: {
-      filters: [
+  grounding: buildDocumentGroundingConfig(
```

Comment:
Same updates can be applied to the [sample app](https://github.com/SAP/ai-sdk-js/blob/main/sample-code/src/orchestration.ts#L232)

Reply:
Done